src/traces/scattermapbox/convert.js

```diff
@@ -20,7 +22,8 @@ var convertTextOpts = require('../../plots/mapbox/convert_text_opts');
 
 var COLOR_PROP = 'circle-color';
 var SIZE_PROP = 'circle-radius';
-
+var OPACITY_PROP = 'circle-opacity';
+var DESELECTDIM = 0.2;
```

Comment:
Ha. You caught being lazy. https://github.com/plotly/plotly.js/issues/1847 will make this thing configurable, but yeah might as well move it to a constant file for now.